environment.yml

```diff
@@ -16,30 +16,27 @@ dependencies:
       - amazon-braket-sdk==1.37.0
       - amazon-braket-strawberryfields-plugin==1.1.0
       - amazon-braket-algorithm-library==1.3.2
-      - cvxpy==1.1.17
+      - cvxpy==1.3.1
       - dask==2023.1.1
       - ecos==2.0.12
       - ipykernel==6.21.1
       - jax==0.4.2
       - jinja2==3.1.2
-      - keras==2.11.0
+      - keras==2.12.0
       - markupsafe==2.1.2
       - matplotlib==3.6.3
       - mypy-extensions==0.4.3
       - openfermion==1.5.1
       - openfermionpyscf==0.5
       - oqpy==0.1.1
-      - nbconvert==7.2.9
```

Comment:
Why remove altogether?

Reply:
I don't think nbconvert would be required to be pinned now, but I think we pinned it since jupyter installs this anyways causing conflicts. We would just want to make sure that doesn't cause issues with other dependencies now.

Reply:
This is something that Sagemaker already has on the instances. I am not sure I see a compelling use case to also include the package in the kernel, as well.